addons/sourcemod/scripting/saxtonhale.sp

```diff
@@ -1159,11 +1159,18 @@ public void TF2_OnConditionAdded(int iClient, TFCond nCond)
 	if (SaxtonHale_IsValidBoss(iClient))
 	{
 		SaxtonHaleBase(iClient).CallFunction("OnConditionAdded", nCond);
```

Comment:
Not sure why milked code is here when it could've been in base_boss and making use of this call along with particles removed, same to conds removed.

I'll still merge it regardless if this is changed or not.